webapp/src/app/api/Store.ts

```diff
@@ -0,0 +1,57 @@
+/* global globalThis */
```

Comment:
I don't think this file should be in `src/app/api`, because that's a special Next.js place where the API _routes_ are. I think it should go somewhere in `src` instead, maybe a new sub-directory?